modules/host/negotiaterenewcontract.go

```diff
@@ -241,7 +241,7 @@ func (h *Host) managedVerifyRenewedContract(so *storageObligation, txnSet []type
 	}
 	// The WindowEnd for the new file contract must be further in the future
 	// than the WindowEnd for the existing file contract.
-	if fc.WindowEnd <= so.proofDeadline() {
+	if fc.WindowEnd < so.proofDeadline() {
```

Comment:
Yeah, that would be best. But run all of your testing with the `<=`, which means cluttering the code a bit but we need to make sure that the testing is handling the correct case.